en/docs/develop/extend/myaccount/set-up-my-account-in-a-dev-environment.md

```diff
@@ -74,7 +74,7 @@ Follow the steps given below to set up My Account in a development environment.
     4.  Expand **Inbound Authentication Configuration**, and then expand **OAuth/OpenID Connect Configuration**. Click **Edit**.
     5.  Change the **Callback URL** field to reflect the port as 9000 or you can add a regexp as follows.
         ```
-        regexp=(https://localhost:9443/user-portal/login|https://localhost:9000/user-portal/login)
+        regexp=(https://localhost:9443/myaccount/login|https://localhost:9000/myaccount/login)
```

Comment:
Are we going to give sample values instead of the placeholders?

Reply:
yes better to have the sample values as this doc contains instructions on how to run it locally